android/fretboard.md

```diff
@@ -0,0 +1,141 @@
+# Fretboard
+This getting started guide is designed to introduce you to adding A/B experiments to your Android apps using the Fretboard Android Component as we use it at Mozilla. This document summarizes and supplements the official Fretboard documentation.
+
+- [What is Fretboard](#what-is-fretboard)
+- [Integrating Fretboard Into A Project](#integrating-fretboard-into-a-project)
+- [Setting Up An Experiment Source](#experiment-source)
+- [Synchronizing Experiments from the Source](#synchronizing-experiments-from-the-source)
+- [Experiment Filters](#experiment-filters)
+- [Experiment Overrides](#experiment-overrides)
+- [Experiment Schema for Kinto](#experiment-schema-for-kinto)
+- [Fretboard README](https://github.com/mozilla-mobile/android-components/blob/master/components/service/fretboard/README.md)
+- [Fretboard Blog Announcement](https://mozilla-mobile.github.io/android-components/2018/08/10/fretboard.html)
+
+## What is Fretboard
+Fretboard is an open source A/B testing framework. It was designed to replace KeepSafe's Switchboard library and server, which was used in the older Firefox for Android, codenamed "Fennec".
```

Comment:
Can we include why we use Fretboard over other frameworks? It could be a follow-up.